protocol/rpc_header_test.go

```diff
@@ -86,7 +86,7 @@ func TestObRpcHeaderEncodeDecode(t *testing.T) {
 	assert.EqualValues(t, obRpcHeader.SessionId(), newObRpcHeader.SessionId())
 	assert.EqualValues(t, obRpcHeader.TraceId0(), newObRpcHeader.TraceId0())
 	assert.EqualValues(t, obRpcHeader.TraceId1(), newObRpcHeader.TraceId1())
-	assert.EqualValues(t, obRpcHeader.Timeout(), newObRpcHeader.Timeout())
+	assert.EqualValues(t, obRpcHeader.Timeout().Milliseconds(), newObRpcHeader.Timeout()/1e6)
```

Comment:
Why use milliseconds in rpcHeader and use /1e6 in newRpcHeader?

Reply:
edit